tslint.json

```diff
@@ -2,7 +2,7 @@
   "extends": ["tslint-config-airbnb", "tslint-config-prettier"],
   "rules": {
     "interface-name": false,
-    "max-line-length": [true, 80],
+    "max-line-length": [false],
```

Comment:
max-line-length really helps for readability. Use `// tslint:disable-next-line:max-line-length` to disable it specifically

Reply:
This was the error where the linting tools were fighting each other. I will try it again re-enabled and see if it works.